README.md

```diff
@@ -82,17 +82,20 @@ Benchmarks measure memoized value retrieval time using
 run on Ruby 3.0.1, except as indicated below for specific gems. Benchmarks are
 run in GitHub Actions and updated in every PR that changes code.
 
-|Method arguments|**`memo_wise` (0.4.0)**|`memery` (1.4.0)|`memoist`\* (0.16.2)|`memoized`\* (1.0.2)|`memoizer`\* (1.0.3)|
```

Comment:
The [rich diff](https://github.com/panorama-ed/memo_wise/pull/153/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) is much easier to review!